### ISSUE TITLE
[Bug?]: CORS dbAuth no longer working after upgrade to Redwood 4

### ISSUE THREAD

**Issue by schybo** [redwoodjs/redwood#7624]
### What's not working?

👋 Recently upgraded to Rails 4. Previously had CORS dbAuth working successfully. I run my site serverful on AWS via Flightcontrol. After upgrade the login continually redirects to the login page (while returning the 'Welcome Back' success popover).

After checking the application logs it seems like the session token is never getting set.

The first `auth` call returns successfully with a cookie to set

<img width="588" alt="image" src="https://user-images.githubusercontent.com/2762046/218278197-ea1330ea-9cea-4534-9b60-3cab24d341a2.png">

But then the very next call to `auth?getToken` returns a `set-cookie` response header that immediately wipes out the session token I think

<img width="576" alt="image" src="https://user-images.githubusercontent.com/2762046/218278255-19d5dc40-9485-4729-abed-8d88776753bf.png">

But that I believe is because it's not including any cookies in the request header (which I checked an old version of the site and it did include a cookie header

<img width="571" alt="image" src="https://user-images.githubusercontent.com/2762046/218278336-852ecd45-0c6a-4e6a-8b8c-2d6736e05211.png">

After some playing around with it. I had a hunch the include credentials config was not being respected

In the CORS guide you specify to include the config like so:
https://redwoodjs.com/docs/cors#auth-xhr-credentials

<img width="963" alt="image" src="https://user-images.githubusercontent.com/2762046/218278517-51550f0f-e705-41e3-aaef-211b6712d333.png">

But pretty sure that no longer works given how `createDbAuthClient` is generated in Redwood4.

I previously had in my `auth.js` file under web (after following the upgrade guide)

```
import { createDbAuthClient, createAuth } from '@redwoodjs/auth-dbauth-web'

const dbAuthClient = createDbAuthClient()

export const { AuthProvider, useAuth } = createAuth(dbAuthClient)
```

but what you really need (for CORS) is now

```
import { createDbAuthClient, createAuth } from '@redwoodjs/auth-dbauth-web'

const dbAuthClient = createDbAuthClient({
  fetchConfig: { credentials: 'include' },
})

export const { AuthProvider, useAuth } = createAuth(dbAuthClient)
```

As specifying `fetchConfig` from `App.ts` seems to no longer have an effect.

Could just be a matter of updating the guide. Or look into why the config can't be pulled from setting it there?

### Are you interested in working on this?

- [ ] I'm interested in working on this

**Comment by Tobbe** [comment]
Thanks for the issue report @schybo 
Looks like we (I) forgot to update the CORS docs. The correct way to do it now is to set the config when creating the dbAuth client in `auth.js`/`auth.ts`. But it looks like you figured that out already 👍

**Comment by schybo** [comment]
@Tobbe Want me to update the docs? I'm happy to do it today

**Comment by schybo** [comment]
@Tobbe https://github.com/redwoodjs/redwood/pull/7627

**Comment by Tobbe** [comment]
Closed by #7627
